### ISSUE TITLE
cryptic toast messages on the live site

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#891]
Instead of human-readable information displayed in the toast messages to tell a user what to do, we're getting JSON/Syntax/token errors. 

The "correct" messages are displaying on the test site so might be something with EROS blocking the right toast messages?

For example, if I leave comment type empty on the live site and hit "add comment" I get this:
![image](https://user-images.githubusercontent.com/49287076/75187343-7bb7f200-570f-11ea-8604-79d04f8822f2.png)

versus on the test site you see this:
![image](https://user-images.githubusercontent.com/49287076/75187798-7313eb80-5710-11ea-972d-a4aeb1af0a07.png)

Another example is typing in the wrong password on the live site gets you this:
![image](https://user-images.githubusercontent.com/49287076/75187432-a144fb80-570f-11ea-8b37-c11d079907c2.png)

versus on the test site you see this:
![image](https://user-images.githubusercontent.com/49287076/75187721-46f86a80-5710-11ea-9537-8ff3f1ccbf82.png)

Here's one more that isn't human-readable on the live site, though I haven't tried it on the test site. This is when someone without permission to change an event off "complete" status tries to make that change:
![image](https://user-images.githubusercontent.com/49287076/75187856-96d73180-5710-11ea-85ea-cd62ccea81e6.png)

**Comment by BlakeDraper** [comment]
Yes, this is an EROS issue - it seems with some of the errors, the server is for some reason blocking the response. This HTML error page is a tell-tale Apache error screen.
![image](https://user-images.githubusercontent.com/6978380/75188969-d4d55500-5712-11ea-8901-005a23069fd3.png)
